<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiruMainWindowClass</class>
 <widget class="QMainWindow" name="FiruMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="editInput">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 127);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listSources">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>aamu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>kana</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>liha</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>uimahalli</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="prgTask">
      <property name="value">
       <number>24</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>274</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
selection-background-color: rgb(170, 255, 255);</string>
   </property>
  </widget>
  <action name="actionOpenDict">
   <property name="text">
    <string>OpenDict</string>
   </property>
  </action>
  <action name="actionOpenTrainer">
   <property name="text">
    <string>OpenTrainer</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>editInput</tabstop>
  <tabstop>listSources</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>editInput</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FiruMainWindowClass</receiver>
   <slot>setInputWord(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>inputWordChanged(QString)</signal>
  <slot>setInputWord(QString)</slot>
 </slots>
</ui>
